doc/concepts/labels.rst

```diff
@@ -1,7 +1,7 @@
 .. _labels:
 
-Cell labels
-===========
+Labels
```

Comment:
With the earlier reordering of all the cable_cell specific pages at the bottom of the hierarchy, removing 'Cell' from these page titles will make it not obvious that these are Cell (and in fact cable_cell) specific. I initially added them as middle ground between creating another layer in the page hierarchy and having no association in the hierarchy at all.

If you really wish to remove 'Cell', what's a good way to group these subcomponents of the cable cell?

Reply:
I'll make it `Cable cell labels`